frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -126,35 +169,67 @@ const ReviewPage = () => {
               setTranslatedStoryLines={setTranslatedStoryLines}
               numApprovedLines={numApprovedLines}
               setNumApprovedLines={setNumApprovedLines}
+              isReviewable={stage === "REVIEW"}
               reviewPage
             />
           </Flex>
-          <Flex margin="20px 30px" justify="flex-start" alignItems="center">
-            <Box marginRight="10px">
-              <ProgressBar
-                percentageComplete={
-                  (numTranslatedLines / translatedStoryLines.length) * 100
-                }
-                type="Translation"
-              />
-            </Box>
-            <Box>
-              <ProgressBar
-                percentageComplete={
-                  (numApprovedLines / translatedStoryLines.length) * 100
-                }
-                type="Review"
-              />
-            </Box>
+          <Flex margin="20px 30px" justify="space-between" alignItems="center">
+            <Flex justify="flex-start" alignItems="right">
+              <Box marginRight="10px">
+                <ProgressBar
+                  percentageComplete={
+                    (numTranslatedLines / translatedStoryLines.length) * 100
+                  }
+                  type="Translation"
+                />
+              </Box>
+              {/* BUG: Need to make these the same width */}
+              <Box>
+                <ProgressBar
+                  percentageComplete={
+                    (numApprovedLines / translatedStoryLines.length) * 100
+                  }
+                  type="Review"
+                />
+              </Box>
+            </Flex>
+            <Tooltip
+              hasArrow
+              label={REVIEW_PAGE_TOOL_TIP_COPY}
+              isDisabled={stage === "REVIEW"}
+            >
+              <Box>
+                <Button
+                  colorScheme="blue"
+                  size="secondary"
+                  margin="0 10px 0"
+                  width="250px"
+                  disabled={stage === "TRANSLATE"}
+                  onClick={openModal}
+                >
+                  RETURN TO TRANSLATOR
+                </Button>
+              </Box>
+            </Tooltip>
           </Flex>
         </Flex>
         <CommentsPanel
-          disabled={false}
+          disabled={stage === "TRANSLATE"}
           commentStoryTranslationContentId={commentStoryTranslationContentId}
           commentLine={commentLine}
           storyTranslationId={storyTranslationId}
           setCommentLine={setCommentLine}
+          reviewPage
         />
+        {returnToTranslator && (
+          <ConfirmationModal
+            confirmation={returnToTranslator}
+            onConfirmationClick={returnToTranslatorMutation}
+            onClose={closeModal}
+            confirmationMessage={REVIEW_PAGE_RETURN_TO_TRANSLATOR_CONFIRMAITON}
+            buttonMessage={CONFIRM_BUTTON_MESSAGE}
```

Comment:
nit: can you make a string like `TRANSLATION_PAGE_BUTTON_MESSAGE = "I'm sure, send for review"` and use that instead of "Confirm"?